lerna.json

```diff
@@ -4,5 +4,5 @@
   ],
   "npmClient": "yarn",
   "useWorkspaces": true,
-  "version": "0.20.7"
```

Comment:
`0.20.7` should have been overrided? It should be something like `1.0.0-beta.1`? Did we miss anything?

Reply:
Checked with both the release branch we have.. https://github.com/cennznet/api.js/pull/83/files and https://github.com/cennznet/api.js/pull/87.. I think it is expected to be 1.0.0